pkgs/applications/version-management/git-and-tools/git-machete/default.nix

```diff
@@ -16,10 +16,7 @@ buildPythonApplication rec {
 
   nativeBuildInputs = [ installShellFiles pbr ];
 
-  # TODO: Add missing check inputs (2019-11-22):
-  # - stestr
-  doCheck = false;
-  checkInputs = [ flake8 mock pycodestyle pylint tox ];
+  checkInputs = [ flake8 mock pycodestyle pylint stestr tox ];
```

Comment:
Why do we need tox here?

Reply:
tests seems to be excluded in the pypi version, should we switch the the github version or should we just remove the check inputs

Reply:
Just switch to the github download. We do that a lot.

Reply:
@figsoda Are you going to make the change in a new PR? How does this work? Also add on checkPhase, `stestr run`. I forgot this. This PR was part of openstack PR that wasn't merged. Wanted to keep the positive changes. But then, I didn't put too much thought into this package, mostly that `stestr` was missing, and is now available.

Reply:
yes